pageserver/src/tenant/timeline.rs

```diff
@@ -2995,7 +3108,12 @@ impl Timeline {
         for l in new_layers {
             let new_delta_path = l.path();
 
-            let metadata = new_delta_path.metadata()?;
+            let metadata = new_delta_path.metadata().with_context(|| {
+                format!(
+                    "failed to read file metadata for new created layer {}",
```

Comment:
same here, omit the `failed to`

Reply:
[06bf49c](https://github.com/neondatabase/neon/pull/3598/commits/06bf49cb9eafe38f2e95782cf7dcbc4dd28c52e1)